doc/news/changes/minor/20160214MathiasMentler

```diff
@@ -0,0 +1,6 @@
+Fixed: double contraction of two SymmetricTensors<..,VectorizedArray<T>> now
+works. Introduced NumberType<T> with static member ::value to be called instead
```

Comment:
...and write `internal::NumberType` and `NumberType::value` so that doxygen can correctly link to it.